lib/lib-table-cluster/src/main/java/org/sagebionetworks/table/cluster/SQLUtils.java

```diff
@@ -711,7 +711,11 @@ public static String createTableIfDoesNotExistSQL(IdAndVersion tableId, boolean
 			builder.append(ROW_ETAG).append(" varchar(36) NOT NULL, ");
 			builder.append(ROW_BENEFACTOR).append(" BIGINT NOT NULL, ");
 		}
-		builder.append("PRIMARY KEY (").append("ROW_ID").append(")");
+		builder.append("PRIMARY KEY (").append("ROW_ID");
+		if(isView) {
+			builder.append(", ").append(ROW_VERSION);

```

Comment:
We should rollback this change and instead validate that the dataset scope does not contain duplicate object ids. This change would lead to unexpected side effects (multi-values, queries etc) and for now it's better to preserve the original functionality of views.